__tests__/integration/auth.spec.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config()
+const request = require('supertest');
+const server = require("../../server/app");
+const { expect } = require('chai');
+const log = require('loglevel');
+const sinon = require("sinon");
+const chai = require("chai");
+chai.use(require('chai-uuid'));
+const Zaven = require("../mock-data/Zaven.json");
+const testUtils = require("./testUtils");
+
+describe('Authentication', () => {
+  let registeredUser;
+
+  beforeEach(async () => {
+    await testUtils.clear();
+    registeredUser = await testUtils.register(Zaven);
```

Comment:
Use testUtils to prepare the data, support all kinds of cases, register, login, transfer, pending...